ledger/acctupdates.go

```diff
@@ -68,6 +68,15 @@ const baseResourcesPendingAccountsBufferSize = 100000
 // is being flushed into the main base resources cache.
 const baseResourcesPendingAccountsWarnThreshold = 85000
 
+// baseBoxesPendingAccountsBufferSize defines the size of the base boxes pending buffer size.
+// At the beginning of a new round, the entries from this buffer are being flushed into the base boxes map.
+const baseBoxesPendingAccountsBufferSize = 50000
```

Comment:
this parameter looks interesting, is that based on some estimation of byte size of storage?

Reply:
nvm, I just read the estimation in description...